releases/patch-releases.md

```diff
@@ -59,47 +59,57 @@ releases may also occur in between these.
 
 ### 1.19
 
-Next patch release is **1.19.1**.
+Next patch release is **1.19.3**
 
-| Patch Release | Cherry-picks deadline | Target date |
-| --- | --- | --- |
-| 1.19.2 | 2020-09-11 | 2020-09-16 |
-| 1.19.1 | 2020-09-04 | 2020-09-09 |
+End of Life for **1.19** is **2021-09-30**
```

Comment:
Is 1.19 supposed to have EOL while others are TBD?

Reply:
Tim added that in the previous Pr, but i agree we should define the EOL for others

Reply:
Gotcha 👍 also was table formatting change intentional?